script/configs/holesky/M2_deploy_from_scratch.holesky.config.json

```diff
@@ -0,0 +1,58 @@
+{
+  "chainInfo": {
+    "chainId": 17000
+  },
+  "multisig_addresses": {
+    "pauserMultisig": "0x53410249ec7d3a3F9F1ba3912D50D6A3Df6d10A7",
+    "communityMultisig": "0xCb8d2f9e55Bc7B1FA9d089f9aC80C583D2BDD5F7",
+    "operationsMultisig": "0xfaEF7338b7490b9E272d80A1a39f4657cAf2b97d",
+    "executorMultisig": "0x28Ade60640fdBDb2609D8d8734D1b5cBeFc0C348",
+    "timelock": "0xcF19CE0561052a7A7Ff21156730285997B350A7D"
+  },
+  "numStrategies": 2,
+  "strategies": {
+    "numStrategies": 2,
+    "MAX_PER_DEPOSIT": 115792089237316195423570985008687907853269984665640564039457584007913129639935,
+    "MAX_TOTAL_DEPOSITS": 115792089237316195423570985008687907853269984665640564039457584007913129639935,
+    "strategiesToDeploy": [
+      {
+        "token_address": "0x3F1c547b21f65e10480dE3ad8E19fAAC46C95034",
+        "token_name": "Liquid staked Ether 2.0",
+        "token_symbol": "stETH"
+      },
+      {
+        "token_address": "0x7322c24752f79c05FFD1E2a6FCB97020C1C264F1",
+        "token_name": "Rocket Pool ETH",
+        "token_symbol": "rETH"
+      }
+    ]
+  },
+  "strategyManager": {
+    "init_strategy_whitelister": "",
```

Comment:
make this the ops multisig, I think?

Reply:
https://github.com/Layr-Labs/eigenlayer-contracts/pull/473/commits/4f53c1cdb7159ce3c19cd7a4f113d596dbd2882f